OctoKit/Issue.swift

```diff
@@ -58,25 +104,109 @@ public extension Octokit {
             }
         }
     }
+    
+    /**
+     Fetches an issue in a repository
+     - parameter owner: The user or organization that owns the repository.
+     - parameter repository: The name of the repository.
+     - parameter number: The number of the issue.
+     - parameter completion: Callback for the outcome of the fetch.
+     */
+    public func issue(owner: String, repository: String, number: Int, completion: (response: Response<Issue>) -> Void) {
```

Comment:
> We are talking about obvious mistakes here right?

It can be more subtle than that: consider a hypothetical issue browser that has an issue number text field. The developer might attempt to pass in the value of `UITextField.text`. With an `Int`-typed parameter, the compiler will issue a warning about mismatched types. That will give the developer an opportunity to add error handling or text processing logic to handle perfectly reasonable user input like `#2`.